pkg/resolver/mock/resolver.go

```diff
@@ -5,16 +5,23 @@
 package mock
 
 import (
-	"fmt"
-
 	"github.com/ethersphere/bee/pkg/resolver"
 )
 
 // Assure mock Resolver implements the Resolver interface.
 var _ resolver.Interface = (*Resolver)(nil)
 
+// ErrNotImplemented denotes a function has not been implemented.
+type ErrNotImplemented struct{}
+
+// Error returns the formatted not implemented error.
+func (e ErrNotImplemented) Error() string {
+	return "function not implemented"
+}
```

Comment:
This error type is pretty static. Could it be `var ErrNotImplemented = errors.New("function not implemented")`?